src/components/Banner/Banner.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Button } from 'carbon-components-react';
+import { ArrowRight16 } from '@carbon/icons-react';
+
+import {
+  banner,
+  bannerText,
+  bannerDetails,
+  buttonBanner,
+} from './Banner.module.scss';
+
+const Banner = () => {
+  return (
+    <div aria-label="banner" className={banner}>
+      <p className={bannerText}>
+        <strong>
+          Carbon v10 is in maintenance mode. Support will end on Sept. 30, 2024.
+        </strong>
+        <span className={bannerDetails}>Start using v11 now!</span>
```

Comment:
Do you think this could be a link to https://carbondesignsystem.com/migrating/guide/overview?